extensions/wikia/TemplateClassification/scripts/TemplateClassificationLabeling.js

```diff
@@ -52,39 +52,30 @@ define('TemplateClassificationLabeling',
 		})();
 
 		getTitleStrategy = (function getTitleStrategy() {
-			function addTemplate() {
-				return mw.message('template-classification-edit-modal-title-add-template').escaped();
-			}
-
 			function editType() {
 				return mw.message('template-classification-edit-modal-title-edit-type').escaped();
 			}
 
-			function addTypeBeforePublish() {
-				return mw.message('template-classification-edit-modal-select-type-sub-title').escaped();
+			function chooseType() {
+				return mw.message('template-classification-edit-modal-title-select-type').escaped();
 			}
 
 			return {
-				addTemplate: addTemplate,
+				addTemplate: chooseType,
 				editType: editType,
-				addTypeBeforePublish: addTypeBeforePublish
+				addTypeBeforePublish: chooseType
```

Comment:
I still think that the Labeling extension should be unaware of the places it is called at and provide a clear and self-explenatory interface. Imagine that we'll want to use the extension in 2 new places. Will you add 2 more methods to the interface that just execute the `chooseType` logic or wouldn't it be better for them to call the same method and make it consistent?